src/mods/mod_signup.cpp

```diff
@@ -1766,6 +1771,16 @@ void ModSignup::saveNewUserRecord()
     m_user_record->sInverseColor = m_config->default_color_inverse;
     m_user_record->sStatColor = m_config->default_color_stat;
     m_user_record->sBoxColor = m_config->default_color_box;
+    
+    // Check for User Auto Validation
+    if (m_config->use_auto_validate_users)
+    {
+        m_user_record->iLevel = m_config->default_level;
```

Comment:
Add more defaults for file points if use flag is set
Same with Post Call Ratio and Time limits and Time Outs.